doc/west/index.rst

```diff
@@ -54,8 +163,9 @@ Design Constraints
 West is:
 
 - **Optional**: it is always *possible* to drop back to "raw"
-  command-line tools, i.e. use Zephyr without using West. It may not
-  always be *convenient* to do so, however. (If all of West's features
+  command-line tools, i.e. use Zephyr without using west (although west itself
+  might need to be installed and accessible to the build system). It may not
```

Comment:
meaning, the tools inside Zephyr, that was already `west`, such as `flashing, debug, attach, ...`

Those are now in `west` repo, so in order to have those working, you would need the `west` repo.
But if not using `west` for flashing, debug, etc. I assume you can leave it out, although we might need to ensure CMake can handle it, in case `west` is not present.

Reply:
> I assume you can leave it out

That being the point

Reply:
It depends what you do really. If you are not going to flash or debug, then you definitely not need it. In the future however, you will need it in order to build, since the build system will use `west` to locate the different repositories available for building.

Reply:
question is, if people really want / has a reason to avoid cloning `west`.
And Zephyr needs it for tje build system to locate all required repositories for building, then could we provide an easy to do so way, e.g. arguments to CMake. to allow people to do so.

Simply, not preventing them from doing so.
That is give the option that what `west` returns can simply be provided manually ?

Reply:
> In the future however, you will need it in order to build

Ok.. so today you can still build images with that warning, but not doc. And soon nothing. Meaning, not optional :-p